docs/project_droneworld/about.md

```diff
@@ -1,58 +1,54 @@
+- **Current Tech Lead:** Mohammad Al-Hanoosh [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/mohamdlog)  <!--[<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/)-->
 - **Developers:**
-    - Kaleb Yu [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/kalyus)
-    - Szymon Rostkowski [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/sr259)
-    - Hebron Bekele [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/hebronh) 
+    - Student [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/)
+    - Student [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/)
+    - Student [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/) 
 - **Start Date:** 9th Feb 2023
 - **Adoption Date:** 25 Aug 2023
-- **Technologies Used:**
-    - Tech Stack: Simulation, Testing, Visualization, Autonomous Systems
-    - Key Languages: Python, TypeScript with JavaScript
-    - Key Frameworks: React (Frontend), Flask (Backend APIs), Unreal Engine (Simulation)
-    - Databases: MongoDB 
-    - Critical Dependencies: 
-        - PX4 and Ardupilot
-        - Unreal Engine 
+
+### Technologies and Dependencies
+
+- **Key Languages:** Python, TypeScript, JavaScript
+- **Key Technologies:** React (Frontend), Flask (Backend APIs), Unreal Engine with AirSim (Simulation)
+- **Databases:** MongoDB
+- **Critical Dependencies:** PX4, ArduPilot, Unreal Engine
 - **Type:** Desktop
-- **License:**
+- **License:** [MIT](https://opensource.org/license/mit)
 
 ### User Guide
 
-- The user will configure a test scenario through the web interface by specifying details like the number of drones, their sensors, the mission objectives, and the operating environment.
-- The user define critical safety properties that must be monitored during the test, such as minimum separation distance between drones. These properties are based on your system requirements or regulations.
-- DroneWorld generates a detailed test report analyzing the execution. This includes logs, telemetry data, analysis of property violations, and visualizations of key metrics
+The user sets up a test scenario via the web interface by specifying the number of drones, their sensors, mission objectives, and operating environment. They also define critical safety properties, such as minimum separation distances between drones, based on system requirements or regulations. After the test, Drone World provides a comprehensive report that includes logs, telemetry data, property violation analysis, and visualizations of key metrics.
```

Comment:
It doesn't need to happen immediately, but something to consider as a future Issue or set of Issues is development of a more comprehensive user guide that you can link to from here.